.env.example

```diff
@@ -1,3 +1,5 @@
 DAILY_API_TOKEN=abc
-DEEPGRAM_API_KEY=def
 OPENAI_API_TOKEN=sk-abc
+
```

Comment:
These shoudl be `DAILY_API_KEY` and `OPENAI_API_KEY`.

Reply:
Ty, changed; I am surprised by the `load_dotenv()` requirement since it always loads by default on my machine!

Reply:
maybe I added the `load_dotenv()` comment before realizing the variable names were wrong... But I'm surprised that `.env` is loaded without doing anything. unless there's another library that loads it...?

Reply:
But then we wouldn't have it in our `requirements.txt`... I use [direnv](https://direnv.net/) (which is great btw) in my machine which loads `.env` automatically and detects changes... maybe you use the same?